src/autoscaler.ts

```diff
@@ -14,6 +15,29 @@ export interface AutoscaleProcessorOptions {
     redisClient: Redis.Redis;
 }
 
+const groupsManaged = new promClient.Gauge({
+    name: 'autoscaling_groups_managed',
```

Comment:
Will these names cause collisions in the future or do labels take care of it? I feel like even with labels we may want to say "jibri_autoscaling_groups_managed" or something more specific.

Reply:
This is the total number of groups being managed by the service regardless of type.  We may have another stat which has the label of 'type' which could show the count of jibri groups vs. JVB groups or jigasi groups, etc.